src/cook-web/src/components/preview/index.tsx

```
@@ -1,109 +1,113 @@
-                            ))
-                        }
-                    </div>
+        <div className="py-4">
+          <div className="grid grid-cols-2 gap-4">
+            <h3 className="mb-1 text-sm font-medium">出现变量:</h3>
+            <h3 className="mb-1 text-sm font-medium">输入变量值:</h3>
+            {profileItemDefinations.map((field) => (
+              <>
+                <Input value={field.profile_key} />
+                <Input
+                  value={formValues[field.profile_key] || ''}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)